lib/b2.js

```diff
@@ -6,6 +10,14 @@ function B2(options) {
     this.authorizationToken = null;
     this.apiUrl = null;
     this.downloadUrl = null;
+    /*
+        allows an optional retryConfig object that overrides the default retry behaviour
+        please see https://github.com/softonic/axios-retry for additional options
+        retries 3 times by default
+        retries only on network errors and 5xx errors on indempotent requests (GET, HEAD, OPTIONS, PUT, or DELETE) by default
+    */
+    axiosRetry(axios, _.assign({ retries: 3 }, _.get(options, 'retryConfig', {})));
```

Comment:
I'm hesitant to add all of `lodash` just for this one line. Since we have `node >= 10` we can simply use `Object.assign` here:

```js
axiosRetry(axios, Object.assign({ retries: 3 }, options.retryConfig));
```

(the `_.get` is unneeded as Object.assign works with `undefined`)

Reply:
Also, we should probably create a new Axios instance here (`axios.create`), so `axios-retry` isn't injected into the user's code if they also use Axios.